app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/DeleteTask.java

```diff
@@ -106,14 +108,14 @@ protected void onProgressUpdate(String... values) {
       // delete file from media database
       if (!file.isSmb()) {
         try {
-          deleteFromMediaDatabase(cd, file.getPath());
+          deleteFromMediaDatabase(cd.get(), file.getPath());
```

Comment:
Stop doing this, `cd.get()` could return null unpredictably, if you need context during the background operations you need a reference to the application context.